src/location/README.md

```diff
@@ -0,0 +1,15 @@
+# Retail Demo Store location Service
+
+The location web service provides an API for retrieving store locations and customer routes to support to location NRF demo.  
```

Comment:
Minor/nit: "NRF" is no longer relevant to the integration.

Reply:
Expunged everywhere I could find it.